script.py

```diff
@@ -41,42 +42,53 @@ def choose_spell(class_name):
         # Select random spell
         randomSpell = filteredData[random.randint(0, len(filteredData) - 1)]
         # Print random spell
-        return randomSpell['name']
+        return Color.GREEN + randomSpell['name'] + Color.END + "\n" + textwrap.fill(randomSpell['desc'], width=100)
 
 
 # Print out list of all SpellClass variables
 # Then take user input for class
 # Return class name
 def choose_class():
-    print("Bard")
-    print("Cleric")
-    print("Druid")
-    print("Paladin")
-    print("Ranger")
-    print("Sorcerer")
-    print("Warlock")
-    print("Wizard")
+    print("+ " + "Bard - An inspiring magician")
+    print("+ " + "Cleric - A priestly champion")
+    print("+ " + "Druid - A priest of the Old Faith")
+    print("+ " + "Paladin - A holy warrior")
+    print(
+        "+ " + "Ranger - A warrior")
+    print("+ " + "Sorcerer - A spellcaster")
+    print("+ " + "Warlock - A wielder of magic")
+    print("+ " + "Wizard - A scholarly magic-user")
     print()
-    class_name = input("\n" + Color.BOLD + Color.UNDERLINE + "Choose a class: " + Color.END)
+    class_name = input(Color.BOLD + Color.UNDERLINE + "Choose a class:" + Color.END + " ")
```

Comment:
Make sure to keep the space here after the input message 😄 
```suggestion
    class_name = input(Color.BOLD + Color.UNDERLINE + "Choose a class: " + Color.END + " ")
```